core/app/models/spree/reimbursement_type/reimbursement_helpers.rb

```diff
@@ -35,7 +35,13 @@ def create_refund(reimbursement, payment, amount, simulate)
         reason: Spree::RefundReason.return_processing_reason
       })
 
-      simulate ? refund.readonly! : refund.save!
+      if simulate
+        refund.readonly!
+      else
+        refund.save!
+        refund.perform!
```

Comment:
We should use `create_and_perform!` here right?  Otherwise a failed `perform!` could leave an invalid refund in the DB.